docs/testplan/MPLS-test-plan.md

```diff
@@ -0,0 +1,177 @@
+- [Overview](#overview)
+  * [Scope](#scope)
+  * [Testbed](#testbed)
+- [Setup configuration](#setup-configuration)
+    + [Setup of DUT switch](#setup-of-dut-switch)
+- [Test cases](#test-cases)
+  * [Test case \#1 - POP Label](#test-case-1---pop-label)
+    + [Test objective](#test-objective)
+    + [Test steps](#test-steps)
+  * [Test case \#2 - SWAP Label for single label](#test-case-2---swap-label-for-single-label)
+    + [Test objective](#test-objective-1)
+    + [Test steps](#test-steps-1)
+  * [Test case \#3 - PUSH Label](#test-case-3---push-label)
+    + [Test objective](#test-objective-2)
+    + [Test steps](#test-steps-2)
```

Comment:
Would you consider to add new push label test for the single label --> multiple labels case

Reply:
Can add in next phase/step. Label stack done in swap for sanity check.